### PR TITLE
Added Footprint related items

### PR DESCRIPTION
**Pull request by AoShinRO**
* **Addressed Issue(s)**: 

* **Server Mode**: 

* **Description of Pull Request**: missing Red Star Footprint, i can`t find it in kro iteminfo or any rodb

https://divine-pride.net/database/search?q=footprint

2024-04-17+ client needed

https://github.com/user-attachments/assets/465729e2-a34c-4853-9233-12d178db8048

### CODE REVIEW

File: db/re/item_db_etc.yml [hunk 1]
```diff
@@ -89439,7 +89470,12 @@ Body:
     AegisName: aegis_1001615
     Name: Footprint Effect (Garment)
     Type: Etc
-    Weight: 100
+    Weight: 10
```
Comment (Atemo) [inline db/re/item_db_etc.yml:89473]: `#   Weight                  Item weight. Each 10 is 1 weight. (Default: 0)`
```suggestion
    Weight: 100
```

File: db/re/item_db_etc.yml [hunk 2]
```diff
@@ -89439,7 +89470,12 @@ Body:
...
+  - Id: 1001617
+    AegisName: PurpleStar_Foot_Robe
+    Name: Purple Star Footprint (Garment)
+    Type: Etc
+    Weight: 10
```
Comment (Atemo) [inline db/re/item_db_etc.yml:89478]:
```suggestion
    Weight: 100
```

File: db/re/item_db_etc.yml [hunk 3]
```diff
@@ -89709,6 +89745,11 @@ Body:
     Weight: 10
     Flags:
       BuyingStore: true
+  - Id: 1001650
+    AegisName: YellowStar_Foot_Robe
+    Name: Yellow Star Footprint (Garment)
+    Type: Etc
+    Weight: 10
```
Comment (Atemo) [inline db/re/item_db_etc.yml:89752]:
```suggestion
    Weight: 100
```

File: db/re/item_db_etc.yml [hunk 4]
```diff
@@ -90322,6 +90363,11 @@ Body:
       NoGuildStorage: true
       NoMail: true
       NoAuction: true
+  - Id: 1001772
+    AegisName: aegis_1001772
+    Name: Puppy Footprint (Garment)
+    Type: Etc
+    Weight: 10
```
Comment (Atemo) [inline db/re/item_db_etc.yml:90370]:
```suggestion
    Weight: 100
```

File: db/re/item_db_usable.yml [hunk 5]
```diff
@@ -78855,6 +78855,13 @@ Body:
       NoAuction: true
     Script: |
       getgroupitem(IG_STOVE_CHANNELING_BOX_4);
+  - Id: 103263
+    AegisName: Summer_Select_Box4
+    Name: Star Footprint Selection Box (Summer Beach)
+    Type:  Usable
+    Weight: 1
+#   Script: |
+#     /* TODO */
```
Comment (Atemo) [inline db/re/item_db_usable.yml:78864]:
```suggestion
    Weight: 10
```
The item selection box does not require a call from a script command. The item will open the UI using the items defined in `item_packages.yml` and by the client

File: db/re/item_db_etc.yml (hunk 1 above)
Comment (AoShinRO) [reply db/re/item_db_etc.yml:89473]: item info from client say 10
"무게 : ^777777 10 ^000000"

**Pull request merged**